src/reanimated2/core.js

```diff
@@ -15,14 +15,13 @@ if (global._setGlobalConsole === undefined) {
   };
 }
 
+const testWorklet = () => {
+  'worklet';
+};
+
 export const checkPluginState = () => {
-  if (
-    !(() => {
-      'worklet';
-    }).__workletHash &&
-    !process.env.JEST_WORKER_ID
-  ) {
-    throw new Error(
+  if (!testWorklet.__workletHash && !process.env.JEST_WORKER_ID) {
```

Comment:
Unrelated to this PR, but is it possible to expose this in a public API? My use case is to do feature detection regarding if I can use the Reanimated 2 APIs and then use the new implementation if possible.